src/juju/index.js

```diff
@@ -44,14 +43,15 @@ function generateConnectionOptions(usePinger = false, bakery, onClose) {
     closeCallback: onClose,
     debug: false,
     facades,
+    wsclass: WebSocket,
```

Comment:
The old jujulib defaulted to the window.WebSocket whereas the new one requires one passed in to allow for better testing infrastructure.